aries-site/src/examples/components/textinput/TextInputValidationExample.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Box, Button, Form, FormField, TextInput } from 'grommet';
+
+export const TextInputValidationExample = () => (
+  <Box>
+    <Box width="small">
+      <Form>
+        <FormField
+          name="required-field"
+          label="Label"
+          htmlFor="required-field"
+          required
+        >
```

Comment:
I have realized that we can force the error state by providing `error="Error message here"`. This would allow us to immediately show the error state without user interaction, but then loses the interactive demonstration. I'm not sure which way we'd like to go.  I do think we could implement in Grommet Designer file with this approach to not need the button.

Reply:
Yes, I like that for the Designer, I also like it potentially for the site as well from a 'skim and see', but from a "copy and paste" dev perspective, it may cause confusion once it's in a local environment. Thoughts?

Reply:
I think we should pose the question and solicit feedback from the group.

Reply:
Would it work to wire up a lightweight `useState()` in the example, such that the caller could type something to clear the error?

Reply:
Let me try that out.

Reply:
https://deploy-preview-672--keen-mayer-a86c8b.netlify.app/components/select#validation

Reply:
Digging this approach.

Reply:
Let me apply that to this.